core/src/main/java/org/geysermc/geyser/configuration/GeyserJacksonConfiguration.java

```diff
@@ -270,6 +270,15 @@ public AuthType authType() {
             return authType;
         }
 
+        @Setter
+        @JsonProperty("resolve-srv")
+        private boolean resolveSrv = false;
```

Comment:
Not sure if we even want to allow this to be configurable, if not I'll remove this and just have the override to return false.